pyproject.toml

```diff
@@ -17,10 +16,21 @@ requests = "^2"
 click = "^8"
 pydantic = "<2"
 websocat = "^1.12.0"
-golem-node = "0.12.2"
+golem-node = "0.12.3"
 
 setuptools = "*"
 
+[tool.poetry.group.ray.dependencies]
+python = "^3.8"
+ray = {version="^2.7.1", extras=["default"]}
+# golem-api-python = { path="../golem-core-python", develop=true }
+golem-api-python = { git="https://github.com/golemfactory/golem-core-python.git", branch="main" }
```

Comment:
What do we need `golem-api-python` on ray for?

Reply:
that's one of the dependencies of `ray-on-golem`...

Reply:
Unfortunately, in current state imports requires golem core event for running inside of the provider.